lib/fathead/celery/output.txt

```diff
@@ -0,0 +1,908 @@
+replace	R	Task.replace										
+Task.retry	A			celery.app.task Functions							<section class="prog__container"><pre><code>Task.retry(args=None, kwargs=None, exc=None, throw=True, eta=None, countdown=None, max_retries=None, **options)</code></pre><p>Retry the task.</p><span class="prog__sub">Example</span><pre><code>>>> from imaginary_twitter_lib import Twitter>>> from proj.celery import app>>> @app.task(bind=True) def tweet(self, auth, message):     twitter = Twitter(oauth=auth)     try:         twitter.post_status_update(message)     except twitter.FailWhale as exc:         # Retry in 5 minutes.         raise self.retry(countdown=60 * 5, exc=exc)</code></pre><span class="prog__sub">Parameters</span><ul class="prog__ul"><li>args (Tuple) – Positional arguments to retry with.</li><li>kwargs (Dict) – Keyword arguments to retry with.</li><li>exc (Exception) – Custom exception to report when the max restart</li><li>limit has been exceeded (default:</li><li>MaxRetriesExceededError).</li><li>If this argument is set and retry is called while</li><li>an exception was raised (sys.exc_info() is set)</li><li>it will attempt to re-raise the current exception.</li><li>If no exception was raised it will raise the exc</li><li>argument provided.</li><li>countdown (float) – Time in seconds to delay the retry for.</li><li>eta (dateime) – Explicit time and date to run the</li><li>retry at.</li><li>max_retries (int) – If set, overrides the default retry limit for</li><li>this execution.  Changes to this parameter don’t propagate to</li><li>subsequent task retry attempts.  A value of None,</li><li>means “use the default”, so if you want infinite retries you’d</li><li>have to set the max_retries attribute of the task to</li><li>None first.</li><li>time_limit (int) – If set, overrides the default time limit.</li><li>soft_time_limit (int) – If set, overrides the default soft</li><li>time limit.</li><li>throw (bool) – If this is False, don’t raise the</li><li>Retry exception, that tells the worker to mark</li><li>the task as being retried.  Note that this means the task</li><li>will be marked as failed if the task raises an exception,</li><li>or successful if it returns after the retry call.</li><li>**options (Any) – Extra options to pass on to apply_async().</li></ul><span class="prog__sub">Raises</span><p>celery.exceptions.Retry – To tell the worker that the task has been re-sent for retry.This always happens, unless the throw keyword argumenthas been explicitly set to False, and is considerednormal operation.</p></section>	http://docs.celeryproject.org/en/stable/reference/celery.app.task.html#celery.app.task.Task.retry
```

Comment:
@Lord-Lavios looks like we're creating an `<li>` for each line of text, rather than for each parameter? Is this intentional?